lib/aio/drivers.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (C) 2024 Red Hat, Inc.
```

Comment:
This is only used in one place -- could it be merged into abc.py or jobcontext.py? That'd avoid a lot of boilerplate.

Reply:
It can't be merged into `abc.py` (due to the cyclical dependencies that would result) but it can definitely be merged into `jobcontext.py` if you feel strongly about it.

Reply:
Done.